settings/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	extends: 'plugin:@wordpress/eslint-plugin/recommended',
+
+	rules: {
+		'jsdoc/require-param-type': 0,
```

Comment:
This disables the requirement to have types declared on all function parameters. I've decided to disable for now as it would be a significant effort. We could add these at a later date to better document what's expected.